src/main/scala/io/iohk/ethereum/utils/Config.scala

```diff
@@ -39,11 +41,16 @@ object Config {
     object Peer {
       private val peerConfig = networkConfig.getConfig("peer")
 
-      val connectRetryDelay = peerConfig.getDuration("connect-retry-delay").toMillis.millis
-      val connectMaxRetries = peerConfig.getInt("connect-max-retries")
-      val disconnectPoisonPillTimeout = peerConfig.getDuration("disconnect-poison-pill-timeout").toMillis.millis
-      val waitForStatusTimeout = peerConfig.getDuration("wait-for-status-timeout").toMillis.millis
-      val waitForChainCheckTimeout = peerConfig.getDuration("wait-for-chain-check-timeout").toMillis.millis
+      val connectRetryDelay: FiniteDuration = peerConfig.getDuration("connect-retry-delay").toMillis.millis
+      val connectMaxRetries: Int = peerConfig.getInt("connect-max-retries")
+      val disconnectPoisonPillTimeout: FiniteDuration = peerConfig.getDuration("disconnect-poison-pill-timeout").toMillis.millis
+      val waitForStatusTimeout: FiniteDuration = peerConfig.getDuration("wait-for-status-timeout").toMillis.millis
+      val waitForChainCheckTimeout: FiniteDuration = peerConfig.getDuration("wait-for-chain-check-timeout").toMillis.millis
+
+      val maxBlocksHeadersPerMessage: Int = peerConfig.getInt("max-blocks-headers-per-message")
```

Comment:
@adamsmo Can we create a trait called `InfoPerMessage` (it's not a good name, lol) and an object here to implement it?

I'm talking about something like https://github.com/input-output-hk/etc-client/blob/phase/3/blockSync/src/main/scala/io/iohk/ethereum/utils/Config.scala#L87